Lib/test/test_ssl.py

```diff
@@ -1345,6 +1346,7 @@ def test_load_verify_cadata(self):
             ctx.load_verify_locations(cadata=b"broken")
 
 
+    @unittest.skipIf(Py_DEBUG, "Crashes on debug python builds")
```

Comment:
This is certainly not a general issue - can we scope it down to the platform(s) where it crashes?

Reply:
I will scope it down.  

I think it's limited to UCRT.  The problem is that SSL has  FILE* from the ucrtbase.dll table and python has a FILE* from the ucrtbase**d**.dll table.  

I think we could limit it to win32+arm or platform.win32_is_iot( ).  Unless there's a way to check if python is using UCRT? I couldn't find one.

Reply:
Oh, it's because of mixing FILE* across debug/release builds? In that case we scope it down to Windows and enhance the skip message to say "Avoid mixing debug/release CRT on Windows"

Reply:
Done.